### PR TITLE
[INFINITY-2301] Verify metrics reporting via correct container ID

### PR DESCRIPTION
**Pull request by NimaVaziri**
Tested this with HDFS (journal-0) and saw the stats being correctly emitted.

Majority of the changes are cosmetic due to pep8 auto styling.

### CODE REVIEW

File: testing/sdk_metrics.py [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
 import shakedown
 
-import sdk_cmd as cmd
+import sdk_cmd
 
 log = logging.getLogger(__name__)
 
 
-def get_metrics(service_name, task_name):
+def get_metrics(service_name, package_name, task_name):
```
Comment (nickbp) [inline testing/sdk_metrics.py]: Could you go with `package_name, service_name, task_name` here? Fits a little better (and also trying to standardize that via PR #1446 )

File: testing/sdk_metrics.py [hunk 2]
```diff
@@ -30,30 +30,45 @@ def get_metrics(service_name, task_name):
...
+
+    app_json = json.loads(app_response.text)
+    if app_json['dimensions']['executor_id'] == executor_id:
+        return app_json['datapoints']
 
     raise Exception("No metrics found")
 
 
-def wait_for_any_metrics(service_name, task_name, timeout):
+def wait_for_any_metrics(service_name, package_name, task_name, timeout):
```
Comment (nickbp) [inline testing/sdk_metrics.py]: `package_name, service_name, task_name, timeout` plz

File: testing/sdk_metrics.py (hunk 1 above)
Comment (NimaVaziri) [reply testing/sdk_metrics.py]: Done

File: testing/sdk_metrics.py (hunk 2 above)
Comment (NimaVaziri) [reply testing/sdk_metrics.py]: Done

**Pull request merged**